subsys/caf/events/module_id_list.ld

```diff
@@ -0,0 +1,6 @@
+module_id_list_all :
+{
+	__start_module_id_list = .;
```

Comment:
Can we use `ITERABLE_SECTION_ROM`? I expect it would be only about changing a section name.

Reply:
From what i can see `ITERABLE_SECTIONS` are intended to work well with structured data type (structs), but we have simple void*. I would need to wrap module_id to struct. Is this what you would like to have here? It would add some noise.

Reply:
ok. let's investigate and do it under a separate task